packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -350,10 +350,6 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<ChainTypes.Cosmos> {
           {
             type: 'cosmos-sdk/MsgWithdrawDelegationReward',
             value: {
-              amount: {
-                amount: bnOrZero(0).toString(), // amount here is required to broadcast, but we are withdrawing ALL rewards
```

Comment:
So is the comment here wrong or was `hdwallet` fixed so that this isn't needed anymore?

Reply:
@cjthompson see my comment below, @DaoDev44 was able to claim with native so at least the payload is valid.

Reply:
I think this was required by `proto-tx-builder` at the time of writing; it's not anymore in `^0.2.1`.